packages/dev/docs/pages/react-spectrum/styling.mdx

```diff
@@ -0,0 +1,204 @@
+  borderRadius="medium"
+  padding="size-250">
+  <TextField label="Name" labelPosition="side" placeholder="John Smith" />
+  <ActionButton marginStart="size-150">Submit</ActionButton>
+</View>
+```
+
+All of the properties supported by `View` are listed below. All style properties covered previously are also supported.
+
+<PropTable component={{
+  props: {
+    // Filter out properties duplicated from StyleProps
+    properties: Object.fromEntries(Object.entries(sharedDocs.exports.ViewStyleProps.properties).filter(([k]) => !sharedDocs.exports.StyleProps.properties[k]))
+  }
+}} links={sharedDocs.links} />
+
+## Color values
+
+Style props that accept colors, such as backgrounds and borders, only accept Spectrum defined color values.
+In addition to ensuring consistency across products, Spectrum colors automatically adapt to color
+scheme changes, e.g. dark mode.
+
+The list of Spectrum color values is visualized below. Use the picker to see how they change between the 
+light and dark color schemes.
+
+```tsx import
+function ColorViewer() {
+  let [selectedKey, setSelectedKey] = useState();
+
+  return (
+    <Example colorScheme={selectedKey}>
+      <Picker label="Color Scheme" selectedKey={selectedKey} onSelectionChange={setSelectedKey}>
+        <Item uniqueKey="light">Light</Item>
+        <Item uniqueKey="dark">Dark</Item>
+      </Picker>
+      <ul style={{columnWidth: 160, listStyle: 'none', padding: 0}}>
+        {console.log(dnaDocs.links[dnaDocs.exports.DimensionValue.id])}
+        {dnaDocs.links[dnaDocs.exports.ColorValue.id].value.elements.filter(v => v.value && !v.value.startsWith('static')).map(v =>
+          <li key={v.value}>
+            <Flex alignItems="center" marginBottom="size-25">
+              <View backgroundColor={v.value} width="size-250" height="size-250" marginEnd="size-100" flexShrink={0} />
+              {v.value}
+            </Flex>
+          </li>
+        )}
+      </ul>
+    </Example>
+  );
+}
+
+ReactDOM.render(<ColorViewer />, document.getElementById('color-viewer'));
+```
+
+<div id="color-viewer" />
+
+## Escape hatches
+
+While we encourage teams to utilize Spectrum design as it is, we do realize that sometimes it is too limited and some
+product specific customizations may be needed. In these cases, we encourage you or your designers to **talk to us**.
+We may be able to suggest an alternative implementation strategy, or perhaps your design can help inform future
+Spectrum additions.
+
+While the traditional `className` and `style` props are not supported in React Spectrum components, there are two
+escape hatches you can **use at your own risk**. These are `UNSAFE_className` and `UNSAFE_style`. Use of these props
+should be considered a **last resort**. They can be used to work around bugs or limitations in React Spectrum,
+but should not be used in the long term.
+
+The reasoning behind this is that `className` and `style` are too flexible. The goal of Spectrum is to unify the 
```

Comment:
Do you think we should say this? It reads like we are saying people can't be trusted 😉. I would make the next paragraph the main reason and if you still want to leave some of this in, tone it back a bit. Like  "If you find the need to use this, then you should get in touch with Spectrum about your use case."